doc/source/serve/doc_code/test_service_pb2.py

```diff
@@ -0,0 +1,85 @@
+# flake8: noqa
```

Comment:
Should we link to or display these `pb2.py` and `grpc.py` files somewhere in the doc? I didn't see if we did

Reply:
Ah makes sense!  That's awesome that we're running the doc code